demisto_sdk/tests/test_files/non-valid-integration-no-test-playbooks.yml

```diff
@@ -0,0 +1,607 @@
+description: Alert and notify users using PagerDuty
+configuration:
+- display: API Key
+  name: APIKey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Service Key (for triggering events only)
+  name: ServiceKey
+  defaultvalue: ""
+  type: 0
+  required: false
+script:
+  feed: True
+  script: |
+    ''' IMPORTS '''
+    import os
+    import json
+    import requests
+    import datetime
+
+    ''' GLOBAL VARS '''
+    # Pagerduty API works only with secured communication.
+    USE_SSL = True
+
+    API_KEY = demisto.params()['APIKey']
+    SERVICE_KEY = demisto.params()['ServiceKey']
+
+    SERVER_URL = "https://api.pagerduty.com/"
+
+    DEFAULT_HEADERS = {
+                          'Authorization': 'Token token=' + API_KEY,
+                          'Accept': 'application/vnd.pagerduty+json;version=2'
+                      }
+
```

Comment:
Same question. Do we really need all this if all we care about is the `tests` field? Are there other validations that need all this other crap?